src/discoverer/discoverer.ts

```diff
@@ -16,6 +17,9 @@ const getExports = (indexFilePath = "src/index.ts"): Record<string, string> => {
     const indexFile = project.getSourceFileOrThrow(indexFilePath);
 
     const exportSymbols = typeChecker.getExportsOfModule(indexFile.getSymbolOrThrow());
+    const firstExportedVariableStatement = exportSymbols.map((e) => e.getEscapedName());
+    logger.info(firstExportedVariableStatement);
+    kill();
```

Comment:
this file is not being used and its disabled. this code is just for testing so no problem leaving the kill() there.
im going back to this function later